src/NativeScript/Metadata/Metadata.mm

```diff
@@ -207,8 +207,10 @@ int compareIdentifiers(const char* nullTerminated, const char* notNullTerminated
     } while (this->_topLevelIndex < this->_globalTable->buckets.count);
 }
 
+extern char startOfMetadataSection __asm("section$start$__DATA$__TNSMetadata");
```

Comment:
Ok, I will double check it.

I think I found a solution, [this guy](http://stackoverflow.com/questions/28978788/crash-reading-bytes-from-getsectbyname) has a similar problem and I will try his solution.